docs/developer-guide/development.md

```diff
@@ -63,7 +63,13 @@ git switch -c MY_BRANCH
 
 #### Building and testing locally
 
-To run the operator locally on your machine (outside a container), invoke the following make target:
+To run the operator locally against a k8s cluster on your machine first you need to fetch dependencies:
+``` bash
+go mod tidy 
+go mod vendor
+```
+
+Then invoke the following make target:
```

Comment:
This is not required, go will automatically fetch the dependencies for any command that needs them. Running `go mod tidy` is only required if changes are made to `go.mod`.